### PR TITLE
Compatible with Emacs 28

### PR DESCRIPTION
**Pull request by wsw0108**

### CODE REVIEW

File: go-dlv.el [hunk 1]
```diff
@@ -108,7 +108,8 @@
     output))
 
 (define-obsolete-variable-alias 'go-dlv-command-name
-  'gud-dlv-command-name)
+  'gud-dlv-command-name
```
Comment (benma) [inline go-dlv.el:111]: can you please explain this change in detail?

Comment (wsw0108) [reply go-dlv.el:111]: Emacs 28 change the signature of `define-obselete-variable-alias` to
```
define-obsolete-variable-alias is a macro defined in byte-run.el.

Signature
(define-obsolete-variable-alias OBSOLETE-NAME CURRENT-NAME WHEN &optional DOCSTRING)
```

`when` is no longer optional.

Comment (benma) [reply go-dlv.el:111]: Please put this into the commit message and change '28' to 'emacs 28' in the commit title.

**Comment by TatriX** [comment]
Hi! Is there anything preventing this from getting merged?

**Comment by benma** [comment]
> Hi! Is there anything preventing this from getting merged?

Not at all, I simply didn't receive a notification because there was no comment after you made the change. I will merge this soon then :+1: thanks for your contribution!

**Comment by TatriX** [comment]
Redirecting thanks to the actual author @wsw0108  :)

**Pull request merged**